### PR TITLE
18 user story about us page 1

### PR DESCRIPTION
**Pull request by SimasJakubenas**
Updated styling to about us page. Ended up not using individual members descpriction. Added socail media icons (no links yet)

### CODE REVIEW

File: about-us.html [hunk 1]
```diff
@@ -17,28 +17,19 @@
...
-        <button class="hamburger">
-            <div class="bar"></div>
-        </button>
     </header>
-    <nav class="mobile-nav">
-        <a href="index.html">FIND CHARGING FAST</a>
-        <a href="https://stef-cruz.github.io/power-up-point/#what-we-do">WHAT WE DO</a>
-        <a href="about-us.html"  class="active" aria-current="page">WHO WE ARE</a>
-        <a href="contact.html">CONTACT US</a>
-    </nav>
```
Comment (GeorginaCarlisle) [inline about-us.html:33]: Everything above here, needs to stay as it is in the main

**Review by GeorginaCarlisle — COMMENTED** [review]
Just issues with the header. Header needs to stay as per the main.

File: about-us.html [hunk 2]
```diff
@@ -56,38 +47,86 @@ <h1>WHO WE ARE</h1>
...
+                    </div>
                     <div class="team-card_text">
                         <h2>Ib</h2>
                     </div>
                 </div>
+                <!-- Simas's card -->
                 <div class="team-card">
-                    <img src="assets/pictures/simas.jpeg" alt="Picture of Simas">
+                    <div class="team-picture-container">
+                        <img src="assets/pictures/simas.jpeg" alt="Picture of Deirdre">
```
Comment (stef-cruz) [inline about-us.html:123]: Alt needs to change in all images

File: about-us.html [hunk 3]
```diff
@@ -56,38 +47,86 @@ <h1>WHO WE ARE</h1>
...
                     </div>
                 </div>
+                <!-- Georgina's card -->
                 <div class="team-card">
-                    <img src="assets/pictures/georgina.jpeg" alt="Picture of Georgina">
+                    <div class="team-picture-container">
+                        <img src="assets/pictures/georgina.jpeg" alt="Picture of Deirdre">
+                        <div class="team-card_info">
+                            <span>Hackathon<br> Enthusiast</span>
+                            <i class="fab fa-github-square github-icon"><a href="#"></a></i>
```
Comment (stef-cruz) [inline about-us.html:98]: could you add the link to people's GH profile please?

**Pull request closed without merging**